libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/RouteLegProgress.kt

```diff
@@ -8,7 +8,7 @@ import com.mapbox.api.directions.v5.models.RouteLeg
  * in the directions route, much of this information will be identical to the parent
  * [RouteProgress].
  *
- * The latest route leg progress object can be obtained through the [com.mapbox.navigation.core.trip.session.RouteProgressObserver].
+ * The latest route leg progress object can be obtained through the [RouteProgressObserver].
```

Comment:
Looks like you forgot to import the RouteProgressObserver class. Also in RouteProgress and RouteStepProgress.

Reply:
> RouteProgressObserver.kt 

:import

Reply:
Same as in https://github.com/mapbox/mapbox-navigation-android/pull/4683#discussion_r682812621